setup.cfg

```diff
@@ -49,7 +49,11 @@ install_requires =
     torch
     transformers
     wandb==0.9.4
-    python-s3 @ https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_core_sci_lg-0.2.4.tar.gz
+    # python-s3 @ https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_core_sci_lg-0.2.4.tar.gz
+    
+dependency_links =
```

Comment:
I read that using "dependency_links" is not recommended (don't remember why) and adding it to install_requires on my local system. Would be interesting to note if this change does indeed fix the build failure.

Reply:
Ah ok good to know. I think that this install of the scispacy model is one of the problems. I'm not sure what the best way is to do this import though. @97harsh uses dependency links like this to install the scispacy package he's using and his build passed, so that's what inspired me

Reply:
I have only installed the en_core_web, since I was unable to install it by putting in requires

Reply:
Looks like this worked in the end :)